geometry_msgs/Pose docking_pose
---